app/src/main/java/org/listenbrainz/android/App.kt

```diff
@@ -64,6 +64,13 @@ class App : Application() {
         }
 
     companion object {
+        fun getApplication(): Application {
```

Comment:
This doesn't look alright. We shouldn't force unwrap. Also, I don't think this should be used anywhere, we should pass the current activity's context directly if need be.

Reply:
Done!!, actually I am using context to create the cache.